pkg/standalone-utils/test/gas/.hardhat-snapshots/WeightedLPOracle - 2 tokens

```diff
@@ -1,3 +1,3 @@
-Min: 92.8k
-Max: 93.2k
-Avg: 93.0k
\ No newline at end of file
+Min: 101.6k
```

Comment:
were these outdated?

We're using `address(0)` as the sequencer uptime interface in the gas tests, so the overhead in this case should be minimal. Can you explain this difference?

Reply:
(just to clarify: I don't see anything obviously wrong with this PR, but the gas shouldn't increase by that much, which makes me think I might be missing something if this diff effectively corresponds to this change)

Reply:
They might just be outdated. I will check. I agree it shouldn't change much. Maybe a little, to do the extra check, but it should immediately return.

Reply:
were these outdated in the end?